IQKeyboardManager.d.ts

```diff
@@ -0,0 +1,302 @@
+
+	static appearanceWhenContainedIn(ContainerClass: typeof NSObject): IQTitleBarButtonItem; // inherited from UIAppearance
+
+	static appearanceWhenContainedInInstancesOfClasses(containerTypes: NSArray<typeof NSObject>): IQTitleBarButtonItem; // inherited from UIAppearance
+
+	static new(): IQTitleBarButtonItem; // inherited from NSObject
+
+	font: UIFont;
+
+	selectableTextColor: UIColor;
+
+	titleInvocation: NSInvocation;
+
+	constructor(o: { title: string; });
+
+	initWithTitle(title: string): this;
+
+	setTitleTargetAction(target: any, action: string): void;
+}
+
+declare class IQToolbar extends UIToolbar implements UIInputViewAudioFeedback {
+
+	static alloc(): IQToolbar; // inherited from NSObject
+
+	static appearance(): IQToolbar; // inherited from UIAppearance
+
+	static appearanceForTraitCollection(trait: UITraitCollection): IQToolbar; // inherited from UIAppearance
+
+	static appearanceForTraitCollectionWhenContainedIn(trait: UITraitCollection, ContainerClass: typeof NSObject): IQToolbar; // inherited from UIAppearance
+
+	static appearanceForTraitCollectionWhenContainedInInstancesOfClasses(trait: UITraitCollection, containerTypes: NSArray<typeof NSObject>): IQToolbar; // inherited from UIAppearance
+
+	static appearanceWhenContainedIn(ContainerClass: typeof NSObject): IQToolbar; // inherited from UIAppearance
+
+	static appearanceWhenContainedInInstancesOfClasses(containerTypes: NSArray<typeof NSObject>): IQToolbar; // inherited from UIAppearance
+
+	static new(): IQToolbar; // inherited from NSObject
+
+	doneImage: UIImage;
+
+	doneTitle: string;
+
+	title: string;
+
+	titleFont: UIFont;
+
+	readonly debugDescription: string; // inherited from NSObjectProtocol
+
+	readonly description: string; // inherited from NSObjectProtocol
+
+	readonly enableInputClicksWhenVisible: boolean; // inherited from UIInputViewAudioFeedback
+
+	readonly hash: number; // inherited from NSObjectProtocol
+
+	readonly isProxy: boolean; // inherited from NSObjectProtocol
+
+	readonly superclass: typeof NSObject; // inherited from NSObjectProtocol
+
+	readonly  // inherited from NSObjectProtocol
+
+	class(): typeof NSObject;
+
+	conformsToProtocol(aProtocol: any /* Protocol */): boolean;
+
+	isEqual(object: any): boolean;
+
+	isKindOfClass(aClass: typeof NSObject): boolean;
+
+	isMemberOfClass(aClass: typeof NSObject): boolean;
+
+	performSelector(aSelector: string): any;
+
+	performSelectorWithObject(aSelector: string, object: any): any;
+
+	performSelectorWithObjectWithObject(aSelector: string, object1: any, object2: any): any;
+
+	respondsToSelector(aSelector: string): boolean;
+
+	retainCount(): number;
+
+	self(): this;
+}
+
+declare var kIQDoneButtonToolbarTag: number;
+
+declare var kIQPreviousNextButtonToolbarTag: number;
+
+declare var kIQUseDefaultKeyboardDistance: number;
```

Comment:
So.... I need to know: how did you make this? Seems like it might be good to have a note in here for future reference, because I’m sure this CocoaPod will change in the future.